code/_onclick/telekinesis.dm

```diff
@@ -104,7 +104,7 @@
 /obj/item/tk_grab/afterattack(atom/target , mob/living/user, proximity, params)//TODO: go over this
 	if(!target || !user)
 		return
-	if(last_throw+3 > world.time)
+	if(last_throw+30 > world.time)
```

Comment:
```suggestion
	if(last_throw + 3 SECONDS > world.time)
```

Reply:
Yep, it's even the OLD version, I should probably just rePR this